assets/scripts/featured-repos.js

```diff
@@ -0,0 +1,72 @@
+(async () => {
+  const url = 'https://godaddy-featured-repos.now.sh';
+
+  const dom = {
+    select: document.querySelector.bind(document)
+  };
+
+  const textColorFromBackgroundColor = color => {
+    if (color.length < 5) {
+      color += color.slice(1);
+    }
+
+    return parseInt(color.replace('#', '0x'), 16) > (0xFFFFFF / 2) ? '#333' : '#fff';
```

Comment:
This line is a little surprising.  Hard coded color values?  I'm coming at this with less context of how you're styling things for the github.io pages, but maybe at least a constant and a variable reference (though I get this is one-off kind of code).

Reply:
That's fair. I haven't put much thought into bundling JavaScript, which is a future project documented in #48, and so these values currently couldn't come from some shared file, but I could assign these to constants now and then wire them up when that happens. I'll do that before merging.